tests/v2/validation/nodescaling/README.md

```diff
@@ -9,15 +11,14 @@ These tests are designed to accept an existing cluster that the user has access
 
 ## Getting Started
 In your config file, set the following:
-```json
-"rancher": { 
-  "host": "rancher_server_address",
-  "adminToken": "rancher_admin_token",
-  "userToken": "your_rancher_user_token",
-  "clusterName": "cluster_to_run_tests_on",
-  "insecure": true/optional,
-  "cleanup": false/optional,
-}
+```yaml
+rancher:
+  host: "rancher_server_address"
+  adminToken: "rancher_admin_token"
+  userToken: "your_rancher_user_token"
```

Comment:
we don't support a userToken, it's causing a lot of confusion, can you remove it.